robot_caps:
  actions:
    - name: "NavigateToPose"
      type: "nav2_msgs/action/NavigateToPose"
      plugin_name: "/navigate_to_pose"
      description: "Navigate the robot to a specified pose in the world."
      interface: "action_client"
      params:
        pose_header_frame_id:
          type: "str"
          value: "map"
  topics:
    - name: "SetCmd_vel"
      type: "geometry_msgs/msg/Twist"
      plugin_name: "/cmd_vel"
      description: "Set robot linear and angular velocities indefinitely until further notice."
      interface: "topic_publisher"
  # builtins:
    # - name: "DontUse"
    #   plugin_name: "DontUse"
    #   class_file: "command_adapter"
    #   description: "Don't use this builtin, it is just a placeholder."
    #   params:
    #     duration: "float"
    #   interface: "builtin"
